### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by trinhtan**

### CODE REVIEW

**Comment by trinhtan** [comment]
image

**Comment by trinhtan** [comment]
ready

**Comment by trinhtan** [comment]
ready

**Comment by trinhtan** [comment]
image

**Comment by trinhtan** [comment]
ready

**Comment by thanhhungchu95** [comment]
commented
gộp 4 commit vào đi nhé

**Comment by trinhtan** [comment]
image

**Comment by trinhtan** [comment]
ready

File: config/locales/en.yml [hunk 1]
```diff
@@ -31,3 +31,25 @@
...
+      add: "Thêm"
+      subject_code: "Mã học phần"
+      subject_name: "Tên học phần"
+      class_list_of_subject: "Danh sách lớp"
+      edit: "Sửa"
+      delete: "Xóa"
+      reckon_link: "X e m"
+      edit_link: "Sửa"
+      delete_checkbox: "Xóa"
+      delete_button: "Xóa"
```
Comment (DuongNT1512) [inline config/locales/en.yml]: end file

**Comment by DuongNT1512** [comment]
commented

**Comment by trinhtan** [comment]
image

**Comment by trinhtan** [comment]
ready

**Comment by thanhhungchu95** [comment]
commented

File: config/locales/en.yml [hunk 2]
```diff
@@ -31,3 +31,25 @@
 
 en:
   hello: "Hello world"
+  layouts: 
+    _menu:
```
[the remark this answers is not in the document]
Comment (trinhtan) [reply config/locales/en.yml]: file em dat ten la _menu.html.erb nhung tach pull ra thi file day nam o pull khac roi a, vs em thay trong sample app phan header hay footer cung la _header _footer .-.

[the remark this answers is not in the document]
Comment (thanhhungchu95) [reply config/locales/en.yml]: Bỏ _ đi, cái đó ko cần đâu

**Comment by thanhhungchu95** [comment]
commented
The last comment ,=}}

**Comment by trinhtan** [comment]
image

**Comment by trinhtan** [comment]
ready

**Pull request merged**